filter_plugins/fluentd_params.py

```diff
@@ -0,0 +1,31 @@
+from ansible.module_utils._text import to_text
+from collections import OrderedDict
+
+def params_to_text(params):
```

Comment:
This name should probably be less generic and more informative or otherwise it might clash with some other custom function